substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/NativeImage.java

```diff
@@ -1621,16 +1640,16 @@ protected int buildImage(List<String> javaArgs, LinkedHashSet<Path> cp, LinkedHa
     /**
      * Resolves and lists all modules given a module path.
      *
-     * @see #callListModules(String, List)
+     * @see #callListModules(String, List, String)
      */
-    private Map<String, Path> listModulesFromPath(String javaExecutable, Collection<Path> modulePath) {
+    private Map<String, Path> listModulesFromPath(String javaExecutable, Collection<Path> modulePath, String upgradeModulePath) {
```

Comment:
Then you would modify this to pass in `List<String> javaArgs` as the second parameter.